### PR TITLE
Updated PropTypes to use prop-types package

### PR DESCRIPTION
**Pull request by kmorales13**

### CODE REVIEW

**Comment by st0ffern** [comment]
Thanks for PR 👍 
Please add `prop-types` to the package deps 😉

File: package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "react-native-inview",
   "description": "Detects if a ScrollView or ListView element is inside the users viewport and reports back on changes.",
+  "version": "0.0.2",
```
Comment (st0ffern) [inline package.json]: Dont add version, i use "semantic release" that will automate the releases, versions, and tags.

**Review by st0ffern — CHANGES_REQUESTED** [review]
prop-types still missing in the package.json dependencies

File: package.json (hunk 1 above)
Comment (kmorales13) [reply package.json]: My bad, that was meant for something else 🙈

**Comment by st0ffern** [comment]
Great thanks for PR

**Pull request merged**